src/tsconfig.json

```diff
@@ -7,12 +7,9 @@
     "noEmit": true,
     "strict": true,
     "noImplicitAny": false,
-    "target": "ES2020"
+    "target": "ES2020",
+    "allowSyntheticDefaultImports": true
```

Comment:
Are there downsides to enabling this globally? If we incorrectly try to use a default import for a modern package which doesn't have a default export, will that cause problems?

Reply:
In my opinion there is no downside, but the opposite it would prevent future issues. This change in the tsconfig.json is matching what Babel is doing.

Reply:
OK, let's go with this for now. The code that Babel generates to enable handle imports from CommonJS modules adds bloat to the generated bundle so I'd like to eventually turn both this type checking option and the Babel compatibility code off.